### PR TITLE
Test AssignUniqueId generatees unique ids

### PR DESCRIPTION
**Pull request by kokosing**
Test AssignUniqueId generatees unique ids

Adding test for b19d3dfa19f973bee2755dc145ce15da58e1444a
("Fix base for counter in AssignUniqueIdOperator"). Without mentioned
commit added test fails.

### CODE REVIEW

File: presto-hive/src/test/java/com/facebook/presto/hive/TestHiveDistributedQueries.java [hunk 1]
```diff
@@ -54,4 +57,22 @@ public void testRenameTable()
     {
         // Hive connector currently does not support table rename
     }
+
+    @Test
+    public void testAssignUniqueId()
+            throws Exception
+    {
+        String lineitemX50 = java.util.stream.IntStream.range(0, 50)
```
Comment (losipiuk) [inline presto-hive/src/test/java/com/facebook/presto/hive/TestHiveDistributedQueries.java]: name it `unionLineitem50Times`

**Comment by losipiuk** [comment]
> These are engine tests and should be in presto-tests, not in the Hive connector.

As I understand the issue with putting them in `ATQ` was the fact that it operates on tiny TPC-H. And due to small data size problem was not reproduced.
We will try to workaround that.

**Comment by kokosing** [comment]
@electrum Changes applied.

**Comment by martint** [comment]
Merged, thanks!

**Pull request closed without merging**